File: tests/fixtures/ecosystems/csharp_comprehensive/Models/Person.cs
using System;

namespace Comprehensive.Models
{
    public class Person
    {
        public string Name { get; }
        public int Age { get; }

        public Person(string name, int age)
        {
            Name = name;
            Age = age;
        }

        public virtual string Greet()
        {
            return $"Hi, I'm {Name}";
        }

        public override string ToString()
        {
            return $"Person(Name={Name}, Age={Age})";
        }
    }
}

File: tests/fixtures/ecosystems/java_comprehensive/classes/Person.java
package comprehensive.classes;

import comprehensive.interfaces.Greetable;

/**
 * Base person class.
 */
public class Person implements Greetable {
    private final String name;
    private final int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    @Override
    public String getGreeting() {
        return String.format("Hi, I'm %s", name);
    }

    public String greet() {
        return getGreeting();
    }

    @Override
    public String toString() {
        return String.format("Person{name='%s', age=%d}", name, age);
    }
}
